Reqnroll/StepInfo.cs

```diff
@@ -12,13 +12,15 @@ public class StepInfo
         public BindingMatch BindingMatch { get; set; }
         public StepInstance StepInstance { get; set; }
 
+        public string PickleStepId { get; }
```

Comment:
Should we make this internal, just like the ScenarioInfo props?

Reply:
Yes.